docs/user/metrics/quantity.md

```diff
@@ -1,53 +1,152 @@
 # Quantity
 
-Used to record a single non-negative integer value.
+Used to record a single non-negative integer value or 0.
 For example, the width of the display in pixels.
 
-> **Note**: Quantities are currently only allowed for GeckoView metrics (the `gecko_datapoint` parameter is present) and thus have only a Kotlin API.
+> **IMPORTANT** If you need to _count_ something (e.g. number of tabs open or number of times a button is pressed) prefer using the [Counter](./counter.md) metric type, which has a specific API for counting things and also takes care of resetting the count at the correct time.
```

Comment:
I think this is good, but cc @Dexterp37 if he agrees.

Reply:
Looks good to me! Sorry for the delay!